### PR TITLE
Update python-igraph references

### PR DESCRIPTION
**Pull request by LiamConnors**
To close https://github.com/plotly/plotly.py/issues/4285

Couldn't find a reason for the ` sudo pip install` references on the original PR, https://github.com/plotly/plotly.py/pull/4289, and it seems to work fine without

### CODE REVIEW

File: doc/unconverted/python/igraph-networkx-comparison.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Plotly's Python library is free and open source! [Get started](https://plot.ly/p
 
 
 #### Comparison
-In this tutorial we plot the same network - the coauthorship network of scientists working on network theory and experiment - first as an `igraph.Graph` object, with the Kamada-Kawai layout, and then as a `networkx.Graph`, with the Fruchterman-Reingold layout. Install the Python libraries with `sudo pip install python-igraph` and `sudo pip install networkx`.
+In this tutorial we plot the same network - the coauthorship network of scientists working on network theory and experiment - first as an `igraph.Graph` object, with the Kamada-Kawai layout, and then as a `networkx.Graph`, with the Fruchterman-Reingold layout. Install the Python libraries with `pip install igraph` and `sudo pip install networkx`.
```
Comment (alexcjohnson) [inline doc/unconverted/python/igraph-networkx-comparison.md:34]: Same Q

**Comment by LiamConnors** [comment]
@alexcjohnson looking closer at those examples that use sudo, they are actually for Python 2 and Plotly.py v3, so probably best not to update them at all. `pip install igraph` won't work for Python 2. I think we can probably just remove these pages altogether, but I need to look into the correct way to remove v3 pages.

**Comment by alexcjohnson** [comment]
ah yep, makes sense. Not sure if it's worth removing those pages... could be worthwhile in fact adding links from them to dash-cytoscape docs as the more modern and powerful solution for the same kind of needs.

**Review by alexcjohnson — APPROVED** [review]
💃 You're still changing the Py2 pages from `python-igraph` to `igraph` - but I think that's OK, really nobody should be using Py2 anymore and recent `igraph` and `python-igraph` are apparently literally the same code.

**Pull request merged**